src/System.Private.DataContractSerialization/src/System/Xml/PrefixHandle.cs

```diff
@@ -178,8 +179,10 @@ public int CompareTo(PrefixHandle that)
             return GetString().CompareTo(that.GetString());
         }
 
-        private bool Equals2(PrefixHandle prefix2)
+        public bool Equals(PrefixHandle prefix2)
         {
+            if (ReferenceEquals(prefix2, null))
+                return false;
```

Comment:
Why not just:
```C#
if (prefix2 == null)
    return false;
```
?

Also, technically this is a behavioral change, in that if prefix2 is null, now we'll return false rather than throwing a NullReferenceException.  And if we haven't hit this, that could be because this is an internal type where it'll never be null, in which case this could be extra (minimal) cost without benefit.

@khdang?